zcash_client_backend/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this library adheres to Rust's notion of
   wrapping protocol-address types, the `Recipient` type now wraps a
   `zcash_address::ZcashAddress`. This simplifies the process of tracking the
   original address to which value was sent.
+- MSRV is now 1.66.0.
```

Comment:
I don't actually know if `tonic-build` also had its MSRV bumped (they are developed in the same workspace, but neither have explicit `rust-version`s), but given that CI is passing I think that means it currently isn't using post-1.66 features. So the only thing we should note here is that the `lightwalletd-tonic` feature has MSRV 1.70.0 (until we change workspace MSRV to 1.70 in a subsequent PR):
```suggestion
- MSRV is now 1.66.0 (except for the `lightwalletd-tonic` feature flag which
  requires 1.70.0).
```